cmd/kube-scheduler/app/server.go

```diff
@@ -58,8 +59,8 @@ import (
 	"k8s.io/klog"
 )
 
-// Option configures a framework.Registry.
-type Option func(framework.Registry) error
+// Option configures a frameworkv1alpha1.Registry.
+type Option func(frameworkv1alpha1.Registry) error
```

Comment:
The length of `frameworkv1alpha1` is a little long, but it only happens when both `k8s.io/kubernetes/pkg/scheduler/framework` and `k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1` are imported in a same file. In other cases, we could still alias `k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1`  to `framework`.